### ISSUE TITLE
question on logic

### ISSUE THREAD

**Issue by tim-shockley** [Ylianst/MeshCentral#4458]
https://github.com/Ylianst/MeshCentral/blob/571d9d3c7b7b724045b64f5ed3c0b96bf55578fe/amtmanager.js#L2272

as I see it, the value of requestedState has no affect in the logic 

<pre>
-----------------------------------------------------------
enbld | rqstd | e==6 && rs==2 | es == 2 | es == 6 | VALUE |
-----------------------------------------------------------
  0   |   0   |       F       |    F    |    F    |   F   |
  0   |   2   |       F       |    F    |    F    |   F   |
  2   |   0   |       F       |    T    |    F    |   T   |
  2   |   2   |       F       |    T    |    F    |   T   |
  6   |   0   |       F       |    F    |    T    |   T   |
  6   |   2   |       T       |    F    |    T    |   T   |
-----------------------------------------------------------
</pre>

**Comment by Ylianst** [comment]
```
var kvm = (
((responses['CIM_KVMRedirectionSAP'].response['EnabledState'] == 6) && (responses['CIM_KVMRedirectionSAP'].response['RequestedState'] == 2))
||
(responses['CIM_KVMRedirectionSAP'].response['EnabledState'] == 2) || (responses['CIM_KVMRedirectionSAP'].response['EnabledState'] == 6)
); 
```

**Comment by Ylianst** [comment]
Fixed, will be in v1.0.75.

**Comment by dinger1986** [comment]
@si458 can close
